openlibrary/i18n/__init__.py

```diff
@@ -165,9 +171,10 @@ def ungettext(s1, s2, _n, *a, **kw):
         return value
 
 def gettext_territory(code):
-    """Returns the territory name in the current locale.
-    """
-    locale = load_locale(web.ctx.get('lang', 'en'))
+    """Returns the territory name in the current locale."""
+    # Get the website locale from the global ctx.lang variable, set in i18n_loadhook
+    lang = web.ctx.lang  
+    locale = load_translations(lang)
     return locale.territories.get(code, code)
```

Comment:
Typo; this one's calling a different function.

```suggestion
    locale = load_locale(web.ctx.lang)
    return locale.territories.get(code, code)
```